tests/conformance/state/cosmosdb/cosmodb.go

```diff
@@ -0,0 +1,138 @@
+package cosmosdb
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"html/template"
+	"os"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+const (
+	defaultTimeout       = 60 * time.Second
+	collectionNamePrefix = "test-coll-"
+	accountEnv           = "COSMODB_ACCOUNT_NAME"
+	masterKeyEnv         = "COSMODB_MASTER_KEY"
+	dbnameEnv            = "COSMODB_NAME"
+)
+
+// nolint:gochecknoglobals
+var (
```

Comment:
I thought we had agreed not to have component specific test code, not?

Reply:
This is the Cosmo DB setup code that we previously had in Go (it does not have tests).

Since we agree to move to Python for test setting up, we should remove it here.

Reply:
This will be refactored as part of #599

Reply:
Unless we plan to do some test setup in Go, I am not sure that we can reuse this part of the code at all. So the only refactoring I can see is deleting and replacing with Python, right?